### PR TITLE
Add property "retracts are closed" between T1 and T2.

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by StevenClontz** [comment]
Context: https://mathoverflow.net/questions/434451/all-retracts-are-closed-and-all-compacts-are-closed/435257#435257

**Comment by prabau** [comment]
First a question.  I can see that the new notion of "Retracts are closed" provides a roundabout way to state the theorem that retracts in a Hausdorff space are closed.  The reverse of this seems a random question asked by D. van der Zypen in 2014 on mo, and revived recently by someone wondering about it on mathse and then reposted on mo.  Showing the reverse is not true allows one to explore some interesting examples, which is all good. But what makes this property particularly interesting? (should really be a question for van der Zypen)

Apart from that, I have to say I am strongly opposed to renaming the properties KC and US to more verbose names.  The concepts were older, but the names KC and US were introduced by Wilansky in 1967 and have since been used in the literature many times.  For two recent examples:
- https://www.sciencedirect.com/science/article/pii/S0166864109003307
- https://arxiv.org/pdf/math/0306082.pdf

and the references cited there.  One could say they are standard names that have not been appropriated to mean something else, and hence no possibility of confusion there.  Furthermore, when glancing at lists of theorems involving a property or in some derivation it's quite a bit easier to follow when RC and US stand out than having to read and parse the corresponding verbose textual version.  See also issue #67.

Regarding "RC" as an alias for "Retracts are compact", that was only used once, as a local alias in the latest [mo question](https://mathoverflow.net/questions/434451), and not even by van der Zypen in his first question. Does not seem enough to make it an official alias. (And btw I did a google books and google scholar search and could not find any mention of "retracts are compacts" or variant being studied as a property in some literature anywhere).)

File: theorems/T000310.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+if:
+  P000003: true
+then:
+  P000101: true
+refs:
+  - mo: 191016
+    name: '"All retracts are closed" as separation axiom'
+---
+
+See {{mo:191016}}.
```
Comment (prabau) [inline theorems/T000310.md:12]: https://math.stackexchange.com/questions/805274 seems a better reference as it gives a proof.

File: properties/P000101.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+aliases:
+  - RC
+refs:
+  - mo: 191016
+    name: '"All retracts are closed" as separation axiom'
+  - mo: 434451
+    name: '"All retracts are closed" and "all compacts are closed"'
+---
+
+Every retract subspace ($A\subseteq X$ with a continuous $f:X\to A$ constant on $A$) is closed.
```
Comment (prabau) [inline properties/P000101.md:14]: "constant on $A$" should probably be something like "fixing every element of $A$".

**Comment by prabau** [comment]
Unrelated to this PR.  Seems that theorem numbers T306 - T309 and T313 - T319 are currently free.  Would it be ok if I use some of them to add a few things?  Want to avoid git conflicts.

**Comment by StevenClontz** [comment]
> First a question. I can see that the new notion of "Retracts are closed" provides a roundabout way to state the theorem that retracts in a Hausdorff space are closed. The reverse of this seems a random question asked by D. van der Zypen in 2014 on mo, and revived recently by someone wondering about it on mathse and then reposted on mo. Showing the reverse is not true allows one to explore some interesting examples, which is all good. But what makes this property particularly interesting? (should really be a question for van der Zypen)

A lot of our recent work has been related to such T1 not T2 spaces. I think upvotes on MO witness sufficient interest for including in pi-Base.

**Comment by StevenClontz** [comment]
> Apart from that, I have to say I am strongly opposed to renaming the properties KC and US to more verbose names.

I'm fine reverting those changes. I had forgotten our conversation in #67. While I stick with my personal preference for verbosity, I agree that usage in the literature is an important consideration for the default name, and they'll still appear as aliases so it's all good.

> Does not seem enough to make [RC] an official alias.

I'm also fine with removing this alias.

---

General note on notability: this PR was an experiment for me to use a branch of the pi-Base to help me investigate an interesting MO question. It helped me reveal an error in an eight-year-old accepted MO thread. I think, given the current level of activity this repo has, that's sufficient for inclusion in the database. We'll have a need for editorial decisions at some point to avoid bloat, but for now my primary concern is correctness.

**Comment by StevenClontz** [comment]
> Unrelated to this PR. Seems that theorem numbers T306 - T309 and T313 - T319 are currently free. Would it be ok if I use some of them to add a few things? Want to avoid git conflicts.

Please do. I purposely left gaps for this purpose. Someday we need to have automated numbering: the PR uses something like `T20221128130678` (my current local timestamp) and is renumbered to the earliest six-digit available theorem when merged into master.

**Comment by prabau** [comment]
> General note on notability: this PR was an experiment for me to use a branch of the pi-Base to help me investigate an interesting MO question. It helped me reveal an error in an eight-year-old accepted MO thread. I think, given the current level of activity this repo has, that's sufficient for inclusion in the database. We'll have a need for editorial decisions at some point to avoid bloat, but for now my primary concern is correctness.

I retract my previous dismissal of this property, which I had written before having looked at the full PR.  There are indeed interesting connections to some of the other properties as you have shown.  And revealing this old error in MO was a nice bonus.

**Comment by StevenClontz** [comment]
I think I've addressed the issues you pointed out, thanks!

**Review by prabau — APPROVED** [review]
For property P99 we could change the alias "Sequential limits are unique" to "Unique sequential limits" to better match the name "US".  But completely up to you.  Otherwise everything looks good.

**Pull request merged**